client/packages/common/src/authentication/api/hooks/usePreference.ts

```diff
@@ -29,7 +27,7 @@ export const usePreference = <T extends PreferenceKey>(...prefs: T[]) => {
   };
 
   return useQuery({
-    queryKey: [keys.preferences(), ...prefs],
```

Comment:
This was the source of the bug -- `keys.preferences()` returns an array, so they key array was `[ [ "preferences"], "etc..." ]` when it should have been `[ "preferences"], "etc..." ]`.

The simplification to use the `usePreferencesGraphQL` API helps reduce this potential confusion.